### PR TITLE
Feature/build basic html

### PR DESCRIPTION
**Pull request by piknikki**
--->  ((are you in the right branch??))
## What does this PR do?
 - feature adding basic html structure and teeny bit of css
## How should it be tested?
 - Not really any need, just be able to see some stuff in the UI

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
 node_modules
+
+.idea/
```
Comment (knees4bees) [inline .gitignore:3]: What is this?

Comment (piknikki) [reply .gitignore:3]: Oh this is a folder that webstorm creates but no one else needs it.

File: src/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,77 @@
+/* *** style guide *** */
+/* font-family: 'Alice', serif; */
+/* font-family: 'Raleway', sans-serif; */
+/*colors:  navy blue (#026B81), turquoise (#03B0AA), pink (#D49C96), dusty rose (#CBBDBE)*/
```
Comment (knees4bees) [inline src/css/styles.css:4]: Best colors ever

File: src/css/styles.css [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+/* *** Grid *** */
+
+/* *** Header *** */
+.header__left {
+
+}
+
+.header__right--text {
```
Comment (knees4bees) [inline src/css/styles.css:25]: Would love to hear more about the reasoning behind this naming scheme

Comment (piknikki) [reply src/css/styles.css:25]: Right! This is BEM -- let's talk about it in our meeting today.

**Review by knees4bees — APPROVED** [review]
Great start on the html, Nikki!

**Pull request merged**